src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + "square, area: "
+                + getArea() + " sq.units, side: "
+                + side
+                + " units, color: "
+                + getColor().toLowerCase();
+    }
+}
+

```

Comment:
I can't do it because I will have an error 
[ERROR] C:\Users\User\IdeaProjects\jv-oop-advanced\src\main\java\core\basesyntax\Square.java:1: File does not end with a newline. [NewlineAtEndOfFile]